node-commons/src/main/java/org/apache/iotdb/commons/client/AsyncBaseClientFactory.java

```diff
@@ -25,15 +25,19 @@
 
 import java.io.IOException;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
 
 public abstract class AsyncBaseClientFactory<K, V> extends BaseClientFactory<K, V> {
 
   private static final Logger logger = LoggerFactory.getLogger(AsyncBaseClientFactory.class);
   protected TAsyncClientManager[] tManagers;
   protected AtomicInteger clientCnt = new AtomicInteger();
+  private static final String THRIFT_THREAD_NAME = "TAsyncClientManager#SelectorThread";
```

Comment:
This is defined by ourselves, we should have the ability to name this selector thread at init.
Instead traverse all thread to modify a name that we give to it.